e2e/studio/playwright.config.ts

```diff
@@ -54,6 +57,69 @@ export default defineConfig({
         process.env.VERCEL_AUTOMATION_BYPASS_SELFHOSTED_STUDIO || 'false',
       'x-vercel-set-bypass-cookie': 'true',
     },
+    launchOptions: {
+      args: [
+        // Security/Sandbox settings (required for CI environments)
```

Comment:
Just want to remove any process that we aren't using to speed up parallel tests and ensure nothing chrome is doing might break anything. This is super overkill